services/core-web/common/components/tailings/AssociatedDams.tsx

```diff
@@ -47,7 +45,12 @@ const AssociatedDams = (props) => {
     history.push(url);
   };
 
-  const columns = [
+  // @ts-ignore
+  // @ts-ignore
+  // @ts-ignore
+  // @ts-ignore
+  // @ts-ignore
```

Comment:
Will one @ts-ignore be okay for this part ?

Reply:
😕 How's I miss that!?  Haha, indeed!